tests/http-swf-engine-event/test.rules

```diff
@@ -0,0 +1 @@
+alert http any any -> any any (msg:"SWF Decode Engine Event Set"; file_data; content:"Any Conent"; sid:1;)
```

Comment:
we don't test the matching of a decoder event here, which is the one of the goals of the PRs right?

Reply:
the event is checked in `test.yaml`

this rule causes the swf file to be inspected which then leads to the anomaly event (`test.yaml`).

Reply:
The event logging is checked, but event rules are not tested.